assets/js/amp-invalid-url-post-edit-screen.js

```diff
@@ -287,14 +287,32 @@ var ampInvalidUrlPostEditScreen = ( function() { // eslint-disable-line no-unuse
 	 * And sets this as the src of the status icon <img>.
 	 */
 	component.handleStatusChange = function handleStatusChange() {
-		const onChange = function( event ) {
+		const setRowStatusClass = function( { row, select } ) {
+			const statusText = select.options[ select.selectedIndex ].innerText.trim();
+
+			if ( statusText ) {
+				row.classList.toggle( 'new', 'New Rejected' === statusText || 'New Accepted' === statusText );
```

Comment:
@johnwatkins0 This will break when a different locale is active. Instead of looking at the `option` text, you should use the option _value_ which is an integer which will not change.